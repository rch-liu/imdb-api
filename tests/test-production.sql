SELECT films.original_title, films.is_adult, ratings.num_votes
FROM ratings, films
WHERE films.runtime_minutes > 1 AND
      films.is_adult = FALSE AND
      ratings.tconst = films.tconst AND
      ratings.num_votes >= 30
ORDER BY ratings.average_ratings ASC
LIMIT 10

