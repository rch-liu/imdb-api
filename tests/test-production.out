           runtime_minutes           |      is_adult    |    num_votes
-------------------------------------+------------------+-----------------
 The Corbett-Fitzsimmons Fight       |           False  |   326
 Miss Jerry                          |           False  |   86
 Le clown et ses chien               |           False  |   189
 Autour d'une cabine                 |           False  |   761
 L'hallucination de l'alchimiste     |           False  |   162
 L'auberge ensorcel\xc3\xa9e         |           False  |   787
 Pauvre Pierrot                      |           False  |   1245
 Le manoir du diable                 |           False  |   2575
(8 rows)